topology/probes/neutron/neutron.go

```diff
@@ -397,14 +399,28 @@ func (mapper *NeutronProbe) OnNodeDeleted(n *graph.Node) {
 func (mapper *NeutronProbe) Start() {
 	go func() {
 		for mapper.client == nil {
-			provider, err := openstack.AuthenticatedClient(mapper.opts)
+			client, err := openstack.NewClient(mapper.opts.IdentityEndpoint)
 			if err != nil {
+				logging.GetLogger().Errorf("failed to create neutron client: %s", err)
+				time.Sleep(time.Second)
+				continue
+			}
+
+			if err = openstack.Authenticate(client, mapper.opts); err != nil {
 				logging.GetLogger().Errorf("keystone authentication error: %s", err)
 				time.Sleep(time.Second)
 				continue
 			}
 
-			client, err := openstack.NewNetworkV2(provider, gophercloud.EndpointOpts{
+			client.HTTPClient = http.Client{
+				Transport: &http.Transport{
+					TLSClientConfig: &tls.Config{
+						InsecureSkipVerify: config.GetBool("agent.topology.neutron.ssl_skip_verify"),
+					},
+				},
+			}
+
```

Comment:
log a Warning() to advice when neutron client is connected in insecure (like our https server)

Reply:
Added warning